src/components/Hero/hero.style.js

```diff
@@ -132,32 +134,39 @@ H.Content = styled.div`
   left: 2em;
   bottom: 2em;
   padding-bottom: 3em;
-
   bottom: 0 !important;
+  background: rgba(0, 0, 0, 0.9);
```

Comment:
the bg and box shadow needs to be removed because it's ruining the linear gradient in H.ImgContainer